.github/pull_request_template.md

```diff
@@ -0,0 +1,36 @@
+<!--- Provide a general summary of your changes in the Title above -->
+
+## Problem
+
+_What problem are you trying to solve?_
+
+
+## Solution
+
+_How did you solve the problem?_x
```

Comment:
There's a typo here with the `_x` in the end which breaks the italic style

Reply:
Thanks, fixed.